components/brave_new_tab_ui/containers/newTab/addSiteTile.tsx

```diff
@@ -16,25 +14,13 @@ interface Props {
   isDragging: boolean
 }
 
-class AddSite extends React.PureComponent<Props, {}> {
-  render () {
-    return (
-      <AddSiteTile
-        onClick={this.props.showEditTopSite}
-        isDragging={this.props.isDragging}
-      >
-        <AddSiteTileImage>
-          <AddSiteTileIcon/>
-        </AddSiteTileImage>
-        <TileTitle>
-          {getLocale('addTopSiteDialogTitle')}
-        </TileTitle>
-      </AddSiteTile>
-    )
-  }
+export default function AddSite({ showEditTopSite, isDragging }: Props) {
```

Comment:
As I was touching everything else, I converted this to a functional component. Hope it's okay.